src/Resources/config/listener.yml

```diff
@@ -69,6 +69,20 @@ services:
         tags:
             - { name: kernel.event_listener, event: kernel.exception, method: onKernelException, priority: 96 }
 
+    contao.listener.header_replay.backend_session:
+        class: Contao\CoreBundle\EventListener\HeaderReplay\BackendSessionListener
+        arguments:
+            - "%contao.security.disable_ip_check%"
+        tags:
+            - { name: kernel.event_listener, event: t42.header_replay, method: onReplay }
```

Comment:
This would look shit, which is why I decided not to do it. `onTerminal42HeaderReplay()` is not a nice method name at all.